types/validator/validator-tests.ts

```diff
@@ -811,7 +811,16 @@ const any: any = null;
     result = validator.isIdentityCard("sample", "zh-TW");
 
     result = validator.isIP("sample");
+    result = validator.isIP("sample", "4" satisfies IPVersion);
     result = validator.isIP("sample", "6");
+    result = validator.isIP("sample", 4);
+    result = validator.isIP("sample", 6);
+    result = validator.isIP("sample", {});
+    result = validator.isIP("sample", { version: "4" } satisfies IsIPOptions);
+    result = validator.isIP("sample", { version: "6" });
+    result = validator.isIP("sample", { version: 4 });
+    result = validator.isIP("sample", { version: 6 });
+    result = validator.isIP("sample", result ? 6 : { version: 6 });
```

Comment:
Same remark here as in one of the other PRs. Code change is valid, but I do not know if this is the usual way of doing tests.

Reply:
I personally think being exhaustive on test is better :)